src/server/string_family.cc

```diff
@@ -1583,7 +1644,9 @@ void StringFamily::Register(CommandRegistry* registry) {
              GetRange)  // Alias for GetRange
       << CI{"SETRANGE", CO::WRITE | CO::FAST | CO::DENYOOM, 4, 1, 1, acl::kSetRange}.HFUNC(SetRange)
       << CI{"CL.THROTTLE", CO::WRITE | CO::DENYOOM | CO::FAST, -5, 1, 1, acl::kClThrottle}.HFUNC(
-             ClThrottle);
+             ClThrottle)
+      << CI{"GETV2", CO::READONLY | CO::FAST, 2, 1, 1, acl::kGet}.HFUNC(GetV2)
```

Comment:
why do you prefer adding temporary commands ?

Reply:
Because my plan is `build v2 fully in parallel and then delete v1`